pkg/parser/options.go

```diff
@@ -30,3 +31,9 @@ func WithAdditionalInstances(additionalInstances map[string]protoreflect.Message
 		parser.additionalInstances = additionalInstances
 	}
 }
+
+func WithAdditionalTaskPropertiesInstances(additionalTaskProperties []*descriptor.FieldDescriptorProto) Option {
```

Comment:
Do you think `Instances` may be unnecessary here?

Reply:
Fixed in 90f840975ab6c24fae4b512d4c32681c3e4ad143